### ISSUE TITLE
Missing labels for links of icons

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/audit-Liz#5]
### URL
https://lizgarciao.github.io/

### Issues
Missing labels for links of icons

### Severity
High

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Screen reader does not announce what the link is for. 

### Recommended fix
Use aria-label

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
    
### Notes
[Understanding Success Criterion 2.4.4: Link Purpose (In Context) | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context#:~:text=Both%20an%20icon%20and%20text%20are%20included%20in%20the%20same%20link)
